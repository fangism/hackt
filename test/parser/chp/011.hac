// "011.hac" -- dies on unknown identifier

defproc one_shot(chan?(bool) L; chan!(bool) R) {
chp {
	R!(WTF)		// WTF is WTF?
}
}

