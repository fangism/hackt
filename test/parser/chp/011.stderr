object "WTF" not found, ERROR!  [5:5..7]
At least one error in expr-list at [5:4..8]
ERROR: at least one error in CHP body.
